README.md

```diff
@@ -48,6 +48,7 @@ The modular IoT Device Client consists of a “base client” and discrete “cl
  Detect feature](https://docs.aws.amazon.com/iot/latest/developerguide/detect-device-side-metrics.html) in the AWS IoT Device Defender service.
 * The client-side [Fleet Provisioning feature](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html) enables you to replace provisional credentials with device-specific ones
   when you onboard a fleet of devices to AWS IoT Core. It creates a device specific certificate and private key, and registers the device on AWS IoT Core.
```

Comment:
Add the hyperlink here: 
"when you onboard a fleet of devices using the [Fleet Provisioning capability] of AWS IoT Core